### ISSUE TITLE
System.Directory: Can't be safely imported

### ISSUE THREAD

**Issue by k-bx** [ghcjs/ghcjs-boot#37]
Please see https://github.com/kazu-yamamoto/logger/issues/97#issuecomment-233822529 for more details

**Comment by rubenmoor** [comment]
Same here.

**Comment by mightybyte** [comment]
I'm also running into this problem.

**Comment by Profpatsch** [comment]
Yes. Is it possible (and sane?) to declare ghcjs directory as `Trustworthy`?

For now we are going to patch out all `Safe` declarations in `fast-logger` in nixpkgs.

**Comment by CharlesTaylor7** [comment]
Having this issue too. What would be involved to fix this bug?
